packages/react-vapor/src/components/table-hoc/utils/TableHooks.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+export const useCustomLayoutEffect = (isLoading: boolean) => {
+    const tableHeaderRef = React.useRef<HTMLTableHeaderCellElement>();
+    const [columnWidth, setColumnWidth] = React.useState<number>();
+
+    React.useLayoutEffect(() => {
+        if (tableHeaderRef.current && !isLoading) {
+            setColumnWidth(tableHeaderRef.current.offsetWidth);
+        }
+    }, [isLoading]);
+
+    const style = {
+        width: `${columnWidth}px`,
+        minWidth: `${columnWidth}px`,
+        maxWidth: `${columnWidth}px`,
```

Comment:
Do you need the min/max width?

Reply:
I did in the previous attempt, but maybe not now